tests/test_commandline.py

```diff
@@ -97,6 +97,13 @@ def test_metacat_auth_export(auth):
 #    with os.popen("metacat auth import", "r") as fin:
 #        data = fin.read()
 
+# need to first create a namespace for the username
+# also test that the second time it's created fails
+def test_initial_namespace_create(auth):
+    os.popen(f'metacat namespace create {os.environ["USER"]}')
```

Comment:
Can we make this one  os.system() instead of  os.popen()?  Otherwise we're leaking a file descriptor...

Reply:
Yes, I changed it. Thanks for catching that.